project.xml

```diff
@@ -120,6 +120,11 @@
         test = "fty_commmon_rest_selftest" >
     </use>
 
+    <use project = "fty-common-mlm" libname = "libfty_common_mlm" header="fty_common_mlm.h"
```

Comment:
IMHO this project should have dependencies on ZMQ/MLM, and there should be no direct use of those frameworks if possible, as we have now all the communication wrapped in common library

Reply:
I removed the dependency on Malamute, but sadly, ZMQ dependency is necessary, because some parts use zhash - see https://github.com/42ity/fty-rest/blob/master/include/db/dbhelpers.h#L62

Hopefully, we will be able to remove the dependency after we finish cleanup of DB code.